lms/templates/dashboard/_dashboard_status_verification.html

```diff
@@ -6,58 +6,39 @@
 %>
 
 %if verification_status == 'approved':
-<li class="status status-verification is-accepted">
-  <span class="title status-title">${_("ID Verification Status")}</span>
+    <li class="status status-verification is-accepted">
+        <span class="title status-title">${_("Verification Status: Approved")}</span>
 
-  <div class="status-data">
-    <span class="status-data-message">${_("Reviewed and Verified")}</span>
-
-    <div class="status-note">
-      <span class="deco-arrow"></span>
-
-      <p>${_("Your verification status is good for one year after submission.")}</p>
-    </div>
-  </div>
-</li>
+        <div class="status-note">
+            <p>${_("Your edX Verification is reviewed and approved. Your verification status is good for one year after submission.")}</p>
```

Comment:
Does this need to be a ```p``` inside of a ```div```? Can it not just be a ```p``` with class status-note```?